Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN curl -O https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip
 RUN echo '444e22ce8ca0f67353bda4b85175ed3731cae3ffa695ca18119cbacef1c1bea0  sdk-tools-linux-3859397.zip' | sha256sum -c
 RUN unzip sdk-tools-linux-3859397.zip
 RUN echo y | $ANDROID_HOME/tools/bin/sdkmanager --update
-RUN echo y | $ANDROID_HOME/tools/bin/sdkmanager 'platforms;android-30'
+RUN echo y | $ANDROID_HOME/tools/bin/sdkmanager 'platforms;android-31'
```

Comment:
To check: we'd need to update to SDK31 by November, as the Play store's usual yearly requirement. Do we need SDK31 for anything in this pull request?

I ask because we'll need to test carefully, updating SDK versions has broken the app in subtle ways previously (like https://github.com/tailscale/tailscale/issues/2293). If we need to do that testing now, we can. We would need to do it by November anyway.

Reply:
The [target version](https://github.com/tailscale/tailscale-android/blob/83bfea18bb56b319ead0d86597d5e021b96804cc/android/build.gradle#L27) is not bumped by this change, only the SDK version is, to to expose newer symbols.